src/main/resources/application-test.yml

```diff
@@ -0,0 +1,16 @@
+
+spring:
```

Comment:
Skoror przedluzyli nam czas to moze to doszlifowac, trzeba podjac decyzje jaka baza bedzie na produkcji i testy zrobic na obrazie tej bazy, h2 jest tylko do testow lokalnych i w specyficznych sytuacjach zachowuje sie inaczej niz tak postgres na przyklad, poczytaj o Testcontainers

Reply:
Pierwsza wyłącza baner Springa w konsoli. Druga pokazuje, że mam za mało snu i wyjaśnia dlaczego nie mam tego property w aplikacji, w której miało być.  :)